ar_exercise/exercise.rb

```diff
@@ -51,16 +65,31 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+  # zanehs_house = Apartment.create(address: "29 Fairhill Lane", monthly_rent: 200, sqft: 3000, num_beds: 3, num_baths: 4)
+  # grandmas_house = Apartment.create(address: "6508 Springbrook Lane", monthly_rent: 400, sqft: 3200, num_beds: 4, num_baths: 6)
+  # random_house = Apartment.create(address: "670 Fairy Lane", monthly_rent: 900, sqft: 2000, num_beds: 2, num_baths: 1)
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
+  # Tenant.create(name: "Zaneh", age: 23, gender: "female", apartment_id: 21)
```

Comment:
You shouldn't use hard coded IDs in a script, as they are subject to change (esp if you re-seed your DB, they may not start at 1). Instead do something like this:

```ruby
Tenant.create(name: "Ajeh", age: 21, gender: "female", apartment: zanehs_house)
# or this works too
Tenant.create(name: "Ajeh", age: 21, gender: "female", apartment_id: zanehs_house.id)
# or even this!
zanehs_house.tenants.create(name: "Ajeh", age: 21, gender: "female")
```